test/test_player.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+# coding: utf-8
```

Comment:
I'd suggest these changes for consistency.

```suggestion
#!/usr/bin/env python
```

Reply:
I applied your suggestion. But it might be a good idea to keep at least all test files to the same standard header. Should i prepare a pr for this?

Reply:
If you want, you can do that! 🥳

I already tried such cleanup efforts before but over time, inconsistencies keep coming back if they are not enforced by CI tools.

Reply:
E.g. there was #1075